velox/dwio/parquet/tests/duckdb_reader/ParquetTableScanTest.cpp

```diff
@@ -35,6 +35,7 @@ class ParquetTableScanTest : public HiveConnectorTestBase {
 
   void SetUp() override {
     HiveConnectorTestBase::SetUp();
+    unregisterParquetReaderFactory();
```

Comment:
The base class `HiveConnectorTestBase::Setup()` in this change registers the native parquet reader.
To test the DuckDB Parquet reader, we need to unregister that and register the DuckDB Parquet Reader.
The DuckDB Parquet reader is going away in https://github.com/facebookincubator/velox/issues/5108.
So this will go away as part of that PR.